basic/.install/index.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs')
+const { replaceInFiles, deploy, writeEnv, getInfo } = require('graphql-boilerplate-install')
+
+module.exports = async ({ project, projectDir }) => {
+  const templateName = 'graphql-boilerplate'
+
+  replaceInFiles(['server/src/index.js', 'server/package.json', 'server/database/prisma.yml'], templateName, project)
+
+  console.log('Running $ prisma deploy...')
+
+  process.chdir('server/')
+
+  await deploy(false)
+  const info = await getInfo()
+
+  process.chdir('../')
+
+  replaceInFiles(['server/src/index.js'], '__PRISMA_ENDPOINT__', info.httpEndpoint)
+
+  console.log(`\
+Next steps:
+  1. Change directory: \`cd ${projectDir}/server\`
+  2. Start local server: \`yarn start\` (you can now open a Playground at http://localhost:4000)
```

Comment:
This is not accurate - where did you copy it from? 

to open the playground we need to run `yarn dev`, which spins up a new dev server for the playground that injects a valid token into the http header

Reply:
I think this was copied from the react-boilerplate, prior to the move to `yarn dev`. Thanks for catching this. I'll update!

Reply:
Actually, this is still the same as the react-fullstack-graphql basic boilerplate, it looks like

Reply:
did you see this: https://github.com/graphql-boilerplates/react-fullstack-graphql/pull/171/files?

Reply:
Oh sorry, looks like that was merged in only recently 🙂

Reply:
We decided to tackle this in another PR 🙂